spec/ffprober/parser_spec.rb

```diff
@@ -1,109 +1,82 @@
 # encoding: utf-8
-require 'spec_helper'
+require "spec_helper"
 
-def shared_specs
-  describe 'format' do
-    it 'should determine the correct filename' do
-      expect(ffprobe.format.filename).to eq('spec/assets/sample video.m4v')
-    end
-
-    it 'should find the correct size' do
-      expect(ffprobe.format.size).to eq('4611')
-    end
+RSpec.describe Ffprober::Parser do
+  context "from_file", if: Ffprober::FfprobeVersion.valid? do
+    let(:ffprobe) { Ffprober::Parser.from_file("spec/assets/sample video.m4v") }
 
-    it 'should find the correct bit_rate' do
-      expect(ffprobe.format.bit_rate).to eq('377950')
-    end
-  end
+    it 'parses the content' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.